### PR TITLE
Fix unit tests by calling TestWidgetsFlutterBinding.ensureInitialized()

### PR DESCRIPTION
**Pull request by collinjackson**
Fixes tests that were broken by flutter/flutter#38464

See also flutter/flutter#39077

### CODE REVIEW

File: packages/firebase_database/test/firebase_list_test.dart [hunk 1]
```diff
@@ -10,6 +10,7 @@ import 'package:mockito/mockito.dart';
 import 'package:test/test.dart';
 
 void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
```
Comment (mehmetf) [inline packages/firebase_database/test/firebase_list_test.dart]: Need to import flutter_test.

File: packages/firebase_messaging/test/firebase_messaging_test.dart [hunk 2]
```diff
@@ -11,6 +11,7 @@ import 'package:platform/platform.dart';
 import 'package:test/test.dart';
 
 void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
```
Comment (mehmetf) [inline packages/firebase_messaging/test/firebase_messaging_test.dart]: Import flutter_test.

**Comment by collinjackson** [comment]
I'll land this one first and then land https://github.com/FirebaseExtended/flutterfire/pull/3 afterwards and then publish all the plugins.

**Comment by mehmetf** [comment]
LGTM

**Pull request merged**